### PR TITLE
tr(style) Move portal js style from bonita web

### PR DESCRIPTION
**Pull request by velcrin**

### CODE REVIEW

File: assembly-looknfeel-css.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <fileSets>
+        <fileSet>
+            <directory>${project.build.directory}/css</directory>
+            <outputDirectory></outputDirectory>
+            <filtered>true</filtered>
+            <includes>
+                <include>**/*.html</include>
+                <include>VERSION</include>
+            </includes>
+        </fileSet>
```
Comment (abirembaut) [inline assembly-looknfeel-css.xml]: I'm not sure that this is required (the homepage and the version files are only in bonita-web look'n'feel, I think).
By the way, we could put the version only in distrib.

File: assembly-looknfeel-css.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <fileSet>
+            <directory>${project.build.directory}/css</directory>
+            <outputDirectory></outputDirectory>
+            <excludes>
+                <exclude>**/*.html</exclude>
+            </excludes>
```
Comment (abirembaut) [inline assembly-looknfeel-css.xml]: same comment as above

File: Gruntfile.js [hunk 3]
```diff
@@ -161,7 +161,11 @@ module.exports = function (grunt) {
         })()
       },
       rules: [
-        // prefix web appliation
+        {
+          from: '^/bonita/portal/themeResource?theme=portal&location=bonita-skin.css',
```
Comment (velcrin) [inline Gruntfile.js]: Also this is not working

File: assembly-looknfeel-css.xml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.build.directory}/css</directory>
+            <outputDirectory></outputDirectory>
+            <filtered>true</filtered>
+        </fileSet>
```
Comment (abirembaut) [inline assembly-looknfeel-css.xml]: I think this fileset can be removed. Since there is no version fille, I think we have nothing to replace.

**Pull request merged**